src/components/CollectionPreviewCard.tsx

```diff
@@ -17,9 +17,9 @@ interface CollectionPreviewCardProps {
 
 interface CollectionPreviewSkeletonProps {
   background: JSX.Element;
-  preview: JSX.Element;
+  // preview: JSX.Element;
   name: JSX.Element;
-  size: JSX.Element | boolean;
+  dropTime: JSX.Element | boolean;
```

Comment:
How can this be boolean? :)

Reply:
I just changed the name, data and type will be updated with update in graphql data.